pkg/rslog/output_builder.go

```diff
@@ -127,24 +127,35 @@ func (b outputBuilder) build(output LogOutputType, logFilePath string) (io.Write
 	case LogOutputStdout:
 		return b.fileSystem.Stdout(), nil
 	case LogOutputFile:
-		return b.createLogFile(logFilePath)
+		return b.resolveLogFile(logFilePath)
 	default:
 		err := fmt.Errorf("The output %q provided for the logging output configuration field isn't supported.", output)
 		return nil, err
 	}
 }
 
-func (b outputBuilder) createLoggingDir(logFilePath string) {
+func (b outputBuilder) resolveLogFile(logFilePath string) (io.Writer, error) {
 	lgr := DefaultLogger()
+	if strings.TrimSpace(logFilePath) == "" {
+		lgr.Infof("Logging output is set to FILE, but no path was provided.")
```

Comment:
just a thought - L154 is also returning a b.fallbackOutput but logging an Error - should this be an error log as well? Or possibly a warn?

Reply:
good point, yeah this should be definitely a warn, I'll update 👍

Reply:
hmm, I see - that's just a direct copy from createLogFile - so maybe fine to leave as-is then. 

I'm fine with leaving it - just a thought!